nginx.conf.template

```diff
@@ -44,8 +44,15 @@ http {
   # can run a BDB cluster.
   server {
     lua_code_cache on;
-    listen BIGCHAINDB_FRONTEND_PORT;
+    listen BIGCHAINDB_FRONTEND_PORT ssl;
     server_name "FRONTEND_DNS_NAME";
+
+    ssl_certificate        /usr/local/openresty/nginx/conf/ssl/cert.pem;
+    ssl_certificate_key    /usr/local/openresty/nginx/conf/ssl/cert.key;
+    ssl_protocols          TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
It seems strange to allow TLSv1.1 but I suspect there's some reason. Can you maybe add a comment about that? Maybe add a link to a relevant reference or article.